### DISCUSSION TITLE
어쩔랭 v2

### DISCUSSION THREAD

**Discussion by chul0721** [assertive-lang/asserlang#51] [Announcements]
## 어쩔랭 v2
어쩔랭 v1에 대한 인기가 사라져가는 이 무렵 어쩔랭 v2에 대한 소식을 전해 드리려 합니다.  
해당 Discussions 탭을 이용하여 어쩔랭 v2에 대한 의견을 자유롭게 남겨주세요.  
또한 문의와 질문 등은 [디스코드 커뮤니티](https://discord.com/invite/nZEEhDKnvb)를 이용해주세요.

**Comment by sangchoo1201** [comment]


**Reply by sangchoo1201** [reply]
제가 정리해둔 스레드가 있더군요. 과거의 저에게 칭찬 드립니다.

**Reply by chul0721** [reply]
아 있었나요
